Makefile

```diff
@@ -92,7 +79,7 @@ clean:
 	-rm -rf _trial_temp
 	-rm -rf docs/api
 	-rm -rf man/\*.1
-	-rm -rf sdist
+	$(MAKE) -f Makefile.packaging clean
```

Comment:
I think you may have to move this up (and add a comment to that effect) as that other makefile will call python imports in its header and generate a bunch of pyc files after the find runs.

This would brake sourcepackage.